### PR TITLE
fix(app_notifications): add toggle notification menu on second click

### PR DESCRIPTION
**Pull request by FussuChalice**
# Description
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by rhahao** [comment]
Task linked: [CU-86c0u82bh Toggle notification menu on second click](https://app.clickup.com/t/86c0u82bh)

File: src/features/app_notification/index.tsx [hunk 1]
```diff
@@ -10,7 +10,11 @@ const AppNotification = () => {
 
   return (
     <>
-      <ButtonIcon onClick={handleOpenNotification}>
+      <ButtonIcon
+        onClick={handleOpenNotification}
```
Comment (rhahao) [inline src/features/app_notification/index.tsx:14]: @FussuChalice: for this change, my idea would be to just toggle the state value. This should be change on the `useAppNotification.tsx` file:

```
const handleToggleNotificationState = async () => {
    setIsMyAssignmentOpen(false);

    setOpen((prev) => !prev);
};
```

Also, could you revert all changes in the `IconButton` component about the background color handling? At this moment, I do not see yet the need to change it.

Thanks.

**Review by rhahao — CHANGES_REQUESTED** [review]

File: src/features/app_notification/index.tsx (hunk 1 above)
Comment (FussuChalice) [reply src/features/app_notification/index.tsx]: Yes, sure.

**Review by rhahao — APPROVED** [review]

**Pull request merged**